CONTIRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Welcome to client-sdk-javascript contributing guide :wave:
+
+Thank you for taking your time to contribute to our JavaScript SDK!
+<br/>
+This guide will provide you information to start your own development and testing.
+<br/>
+Happy coding :dancer:
+<br/>
+
+## Requirements :coffee:
+
+- Node version [10.13 or higher](https://nodejs.org/en/download/) is required
```

Comment:
Is this still true? Just checking because I know we upgraded the node version with which we test

Reply:
Good point. I can match the version to 16 or higher which is consistent with the node version we're using on the CI/CD workflows.